test/sanity/code-smell/test-constraints.py

```diff
@@ -36,7 +36,7 @@ def main():
             constraints = raw_constraints.strip()
             comment = requirement.group('comment')
 
-            is_pinned = re.search('^ *== *[0-9.]+(\\.post[0-9]+)?$', constraints)
+            is_pinned = re.search('^ *== *[0-9.]+(\\.post[0-9]+|a[0-9]+|b[0-9]+)?$', constraints)
```

Comment:
This change (resp. the whole commit bec7451f9ee0deaa4a37932a98eed62089a2e693) is no longer needed.